apps/server/src/utils/fileManagement.ts

```diff
@@ -114,3 +109,20 @@ export async function dockerSafeRename(oldPath: PathLike, newPath: PathLike) {
   await copyFile(oldPath, newPath);
   await unlink(oldPath);
 }
+
+/**
+ * finds potential file index number in our (*) format and increments
+ */
```

Comment:
maybe we can describe a bit better the business decisions here

```suggestion
/**
 * finds potential file index number in our (*) format and increments
 * the number section (*) must be separated from the name by a space
 * @example incrementProjectNumber('test(1).json') -> 'test(1).json'
  * @example incrementProjectNumber('test (1).json') -> 'test(2).json'
 */
```